resources/lang/id/store.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'admin' => [
+        'warehouse' => 'Gudang',
+    ],
+
+    'checkout' => [
+        'cart_problems' => 'Aduh, ada masalah dengan keranjang Anda!',
+        'cart_problems_edit' => 'Klik di sini untuk menyuntingnya.',
+        'declined' => 'Pembayaran dibatalkan.',
+        'error' => 'Terjadi masalah saat menyelesaikan pembayaran :(',
+        'old_cart' => 'Keranjang Anda nampaknya sudah kedaluwarsa dan telah dimuat ulang, silakan coba lagi.',
+        'pay' => 'Bayar via Paypal',
+        'pending_checkout' => [
+            'line_1' => 'Pembayaran sebelumnya telah dimulai tetapi tidak tuntas.',
```

Comment:
Perhaps try something simpler instead for clarity : 

+            'line_1' => 'Anda belum menuntaskan pembayaran sebelumnya.',